src/gatsby-plugin-theme-ui/index.ts

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash'
+
+import preset from '@rebass/preset'
+import Colors from 'utils/colors'
+
+export const fonts = {
+  body: 'Lato, system-ui, sans-serif',
+  heading: 'Alegreya Sans',
+  monospace: 'Fira Mono',
+  serif: 'Spectral, serif',
+  smallcaps: 'Alegreya Sans SC',
+}
+
+export const shadows = {
+  large: '0 0 12px 12px rgba(40, 40, 40, 0.6)',
+  small: '0 0 4px 4px rgba(40, 40, 40, 0.6)',
+}
+
+export const fontSizes = [12, 14, 16, 20, 24, 28, 36, 44]
+
+export const breakpoints = ['45em', '60em', '75em']
+
+const Theme = _.merge(preset, {
+  breakpoints,
+  colors: Colors,
+  fontSizes,
+  fonts,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)